### PR TITLE
Add solution for Exercise 6 (1 more)

### PR DESCRIPTION
**Pull request by spikespaz**
Also fixed package name for EX7.

### CODE REVIEW

**Comment by spikespaz** [comment]
https://youtu.be/96knAM8HwSM

**Comment by spikespaz** [comment]
There is a mistake made in this PR. Check the code to see if you can spot it, then add a comment on the correct line to request changes.

**Comment by spikespaz** [comment]
I have gone ahead and assigned you as the assignee and reviewer for this PR. Also notice that I added a tag on the right side named "enhancement", as this is a "feature contribution" and not a bug report.

File: solutions/src/main/dev/birkett/shared/Exercise6/Main.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package dev.birkett.shared.Exersize6;
```
Comment (Arctic168) [inline solutions/src/main/dev/birkett/shared/Exercise6/Main.java]: Replace Exersize with Exercise please

**Pull request merged**